src/updatedownloader.cpp

```diff
@@ -58,7 +52,7 @@ std::wstring GetUniqueTempDirectoryPrefix()
     return dir;
 }
 
-std::wstring CreateUniqueTempDirectory()
+std::wstring winsparkle::CreateUniqueTempDirectory()
```

Comment:
Is name space designation `winsparkle::` really needed here when already inside the namespace?